modules/021-cni-cilium/templates/agent/podmonitor.yaml

```diff
@@ -1,8 +1,9 @@
+{{- if (.Values.global.enabledModules | has "operator-prometheus-crd") }}
 ---
 apiVersion: monitoring.coreos.com/v1
 kind: PodMonitor
 metadata:
-  name: agent
+  name: cilium-agent
   namespace: d8-monitoring
   {{- include "helm_lib_module_labels" (list . (dict "app" "agent" "prometheus" "main")) | nindent 2 }}
 spec:
```

Comment:
Do you mind also changing the job label value?

Reply:
Do I change every single label in all resources? To be uniform. But that will rolling-restart cilium pods.